sentry/api/sentry.api

```diff
@@ -798,16 +798,18 @@ public final class io/sentry/ProfilingTraceData$JsonKeys {
 
 public final class io/sentry/ProfilingTransactionData : io/sentry/JsonSerializable, io/sentry/JsonUnknown {
 	public fun <init> ()V
-	public fun <init> (Lio/sentry/ITransaction;Ljava/lang/Long;)V
+	public fun <init> (Lio/sentry/ITransaction;Ljava/lang/Long;Ljava/lang/Long;)V
 	public fun equals (Ljava/lang/Object;)Z
 	public fun getId ()Ljava/lang/String;
 	public fun getName ()Ljava/lang/String;
+	public fun getRelativeEndCpuMs ()Ljava/lang/Long;
 	public fun getRelativeEndNs ()Ljava/lang/Long;
+	public fun getRelativeStartCpuMs ()Ljava/lang/Long;
 	public fun getRelativeStartNs ()Ljava/lang/Long;
 	public fun getTraceId ()Ljava/lang/String;
 	public fun getUnknown ()Ljava/util/Map;
 	public fun hashCode ()I
-	public fun notifyFinish (Ljava/lang/Long;Ljava/lang/Long;)V
+	public fun notifyFinish (Ljava/lang/Long;Ljava/lang/Long;Ljava/lang/Long;Ljava/lang/Long;)V
```

Comment:
Technically a breaking change, should we annotate this method or type saying it's internal?